src/stan/variational/families/normal_fullrank.hpp

```diff
@@ -378,7 +378,21 @@ namespace stan {
           eta(d) = stan::math::normal_rng(0, 1, rng);
         eta = transform(eta);
       }
-
+        /**
+         * New: Draw a posterior sample from a normal distribution, and return the log normal density. Constant (d* log 2 pi) is removed.
```

Comment:
Please document according to Stan's standards: https://github.com/stan-dev/stan/wiki/How-to-Write-Doxygen-Doc-Comments

Please remove `"New"`... as soon as this is in, it's no longer new.